api-js/src/loaders/user-affiliations/load-user-affiliations-by-org-id.js

```diff
@@ -0,0 +1,184 @@
+      i18n._(
+        t`Passing both \`first\` and \`last\` to paginate the \`affiliation\` is not supported.`,
+      ),
+    )
+  } else if (typeof first === 'number' || typeof last === 'number') {
+    /* istanbul ignore else */
+    if (first < 0 || last < 0) {
+      const argSet = typeof first !== 'undefined' ? 'first' : 'last'
+      console.warn(
+        `User: ${userId} attempted to have \`${argSet}\` set below zero for: affiliationLoaderByOrgId.`,
+      )
+      throw new Error(
+        i18n._(
+          t`\`${argSet}\` on the \`affiliations\` cannot be less than zero.`,
+        ),
+      )
+    } else if (first > 100 || last > 100) {
+      const argSet = typeof first !== 'undefined' ? 'first' : 'last'
+      const amount = typeof first !== 'undefined' ? first : last
+      console.warn(
+        `User: ${userId} attempted to have \`${argSet}\` set to ${amount} for: affiliationLoaderByOrgId.`,
+      )
+      throw new Error(
+        i18n._(
+          t`Requesting \`${amount}\` records on the \`affiliations\` exceeds the \`${argSet}\` limit of 100 records.`,
+        ),
+      )
+    } else if (typeof first !== 'undefined' && typeof last === 'undefined') {
+      limitTemplate = aql`SORT affiliation._key ASC LIMIT TO_NUMBER(${first})`
+    } else if (typeof first === 'undefined' && typeof last !== 'undefined') {
+      limitTemplate = aql`SORT affiliation._key DESC LIMIT TO_NUMBER(${last})`
+    }
+  } else {
+    const argSet = typeof first !== 'undefined' ? 'first' : 'last'
+    const typeSet = typeof first !== 'undefined' ? typeof first : typeof last
+    console.warn(
+      `User: ${userId} attempted to have \`${argSet}\` set as a ${typeSet} for: affiliationLoaderByOrgId.`,
+    )
+    throw new Error(
+      i18n._(t`\`${argSet}\` must be of type \`number\` not \`${typeSet}\`.`),
+    )
+  }
+
+  let sortString
+  if (typeof last !== 'undefined') {
+    sortString = aql`DESC`
+  } else {
+    sortString = aql`ASC`
+  }
+
+  let filteredAffiliationCursor
+  try {
+    filteredAffiliationCursor = await query`
+    LET affiliationKeys = (FOR v, e IN 1..1 ANY ${orgDBId} affiliations RETURN e._key)
+    LET retrievedAffiliations = (
+      FOR affiliation IN affiliations
+          FILTER affiliation._key IN affiliationKeys
+          ${afterTemplate}
+          ${beforeTemplate}
+          ${limitTemplate}
+          LET orgKey = PARSE_IDENTIFIER(affiliation._from).key
+          LET userKey = PARSE_IDENTIFIER(affiliation._to).key
+          RETURN MERGE(affiliation, { orgKey: orgKey, userKey: userKey })
+    )
+    LET hasNextPage = (LENGTH(
+      FOR aff IN affiliations
+        FILTER aff._key IN affiliationKeys
+        FILTER TO_NUMBER(aff._key) > TO_NUMBER(LAST(retrievedAffiliations)._key)
+        SORT aff._key ${sortString} LIMIT 1
+        LET orgKey = PARSE_IDENTIFIER(aff._from).key
+        LET userKey = PARSE_IDENTIFIER(aff._to).key
+        RETURN MERGE(aff, { orgKey: orgKey, userKey: userKey })
```

Comment:
Only needed in the `retrievedAffiliations` since this function is only used to see if there is something there and return a bool